src/sgb-xi/filenames.ts

```diff
@@ -22,11 +23,13 @@ export const makeAnwendungsreferenz = (
     kassenart
 ].join("");
 
+/** A.k.a "Verfahrenskennung" */
 export const makeDateiname = (
     dateiindikator: FileType,
-    verfahrensVersion: number
+    transferNumber: number
 ) => [
     dateiindikator == "2" ? "E" : "T",
     "PFL",
-    verfahrensVersion.toString().slice(0, 4).padStart(4, "0")
+    "0", // verfahrensversion. Always 0
+    transferNumber.toString().slice(0, 3).padStart(3, "0")
 ].join("");
```

Comment:
found this mistake here. The version is just a single digit and as far as I understood, always 0. (I.e. there was never a new version scheme for the scheme developed).
The rest (3 digits) are the transfer number, a serial number that is to be increased by one for every file.

It is true that this is not really mentioned in the document https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/pflege/technische_anlagen_aktuell_2/TA1_Anhang_3_20170907_oA.pdf but in the example below, it becomes clear that the number is a running number.

Also, it is the same (and properly documented for Sonstige Leistungserbringer.

Reply:
Ok